### DISCUSSION TITLE
[Feature]: Global Map

### DISCUSSION THREAD

**Discussion by matthinc** [immich-app/immich#1693] [Feature Request]
### Feature detail

It would be nice to have a global map that shows the positions of all backed-up photos.

To limit the number of markers, [Marker Clustering](https://github.com/Leaflet/Leaflet.markercluster) could be used.

### Platform

Web

**Comment by kvalev** [comment]
I see that the issue has been closed, was this implemented? I dont see it in the demo instance.

**Comment by fivestones** [comment]
For immich to be a true google photos replacement, this is a crucial feature. I'd love to see it too.

**Comment by jrasm91** [comment]
Implemented in #2355 and released in v1.55.0

**Comment by andrewdunndev** [comment]
This is incredible, thanks to everyone who contributed!
